<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountDialog</class>
 <widget class="QDialog" name="AccountDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Account</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="sofiaGwNameEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>username</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="sofiaGwUsernameEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>realm</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="sofiaGwRealmEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>from-user</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="sofiaGwFromUserEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>from-domain</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="sofiaGwFromDomainEdit"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>password</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="sofiaGwPasswordEdit"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>extension</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="sofiaGwExtensionEdit"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>proxy</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="sofiaGwProxyEdit"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>register-proxy</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="sofiaGwRegisterProxyEdit"/>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>expire-seconds</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QSpinBox" name="sofiaGwExpireSecondsSpin">
     <property name="value">
      <number>60</number>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>register</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QComboBox" name="sofiaGwRegisterCombo">
     <item>
      <property name="text">
       <string>true</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>false</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>register-transport</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QComboBox" name="sofiaGwRegisterTransportCombo">
     <item>
      <property name="text">
       <string>udp</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>tcp</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>retry-seconds</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QSpinBox" name="sofiaGwRetrySecondsSpin">
     <property name="value">
      <number>30</number>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="label_15">
     <property name="text">
      <string>contact-params</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QLineEdit" name="sofiaGwContactParamsEdit"/>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>caller-id-in-from</string>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="label_16">
     <property name="text">
      <string>ping</string>
     </property>
    </widget>
   </item>
   <item row="15" column="1">
    <widget class="QSpinBox" name="sofiaGwPingSpin">
     <property name="value">
      <number>25</number>
     </property>
    </widget>
   </item>
   <item row="16" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <widget class="QComboBox" name="sofiaGwCallerIdInFromCombo">
     <item>
      <property name="text">
       <string>false</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>true</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AccountDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AccountDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
